### PR TITLE
Add test case for changed lined numbers in use-walrus-if

### PR DESCRIPTION
**Pull request by drdavella**
## Overview
*Add (failing) test case for computed line numbers in `use-walrus-if`*

## Details
* We need to make an update so that these change positions are computed properly

### CODE REVIEW

**Comment by drdavella** [comment]
Bumped the unit test timeout since we keep hitting on this for the Python 3.12 tests. See #338.

File: tests/codemods/test_walrus_if.py [hunk 1]
```diff
@@ -181,3 +181,35 @@ def func(y):
                 print("whatever", y)
             """
         self.run_and_assert(tmpdir, input_code, expected_output)
+
+    @pytest.mark.xfail(reason="Need to recompute line numbers to reflect removed lines")
+    def test_multiple_warlus_changes(self, tmpdir):
```
Comment (andrecsilva) [inline tests/codemods/test_walrus_if.py]: warlus -> walrus

**Review by andrecsilva — COMMENTED** [review]
I'm not sure what is the problem here. The current behavior seems to be correct as it reports the node positions of the `if`s in the original code.

**Comment by drdavella** [comment]
@andrecsilva the line locations here affect the position of the comments in CodeTF. In general we frame our comments in terms of the fix rather than the problem and we want the comments to appear on the right side of the split diff, which means they need to be associated with new code positions.

**Review by clavedeluna — APPROVED** [review]

**Pull request merged**